src/helmdeploy.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
-function helmdeploy {
+helmdeploy () {
+  eval $(cat env_vars)
   if [[ "${google_zone}" != "" ]] && [[ "${k8_cluster}" != "" ]]; then
-    eval $(cat env_var_namespace)
-    eval $(cat env_var_tag)
     helm plugin install https://github.com/thomastaylor312/helm-namespace
```

Comment:
We may want to set this to our own forked repo following the DevOps team's best practices.

Reply:
These cannot be taken out.  It needs to source vars on each docker run the container resets after every run. So no persistent vars.